### ISSUE TITLE
could this work on ESP8266?

### ISSUE THREAD

**Issue by vrabi-cv** [klatremis/esphome-for-deye#35]
Hello, since I have got some Wemos D1 mini ESP8266 chips, I would like to try communication on this chip. I tried to upload the yaml but I got error on this lines:
     lambda: |-
       uint16_t value = modbus_controller::word_from_hex_str(x, 0);
Is it due to different chip, or there has to be more changes made inside the program code?

Thanks Martin

**Comment by runesl** [comment]
No, I tried, but esp8266 has too little memory. Buy an esp32 - they're cheap, and code runs succesfully on that.

**Comment by pvprodk** [comment]
I have tried it as well. and as Rune states the ESP8266 has too little memory.
I have created [a cut down version](https://github.com/pvprodk/esp8266-for-deye) that runs on ESP8266, but I would also recommend going for the full experience on ESP32

**Comment by vrabi-cv** [comment]
Thanks for your affords. I run it on ESP32, and it goes perfect. THX.
